bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v18/V18PsbtRpc.scala

```diff
@@ -0,0 +1,25 @@
+package org.bitcoins.rpc.client.v18
+
+import org.bitcoins.core.currency.Bitcoins
+import org.bitcoins.core.protocol.BitcoinAddress
+import org.bitcoins.core.protocol.transaction.{Transaction, TransactionInput}
+import org.bitcoins.core.script.crypto.HashType
+import org.bitcoins.rpc.client.common.Client
+import org.bitcoins.rpc.client.common.RpcOpts.WalletCreateFundedPsbtOptions
+import org.bitcoins.rpc.jsonmodels.{
+  AnalyzePsbtResult
+}
+import org.bitcoins.rpc.serializers.JsonWriters._
+import org.bitcoins.rpc.serializers.JsonSerializers._
+import play.api.libs.json._
+
+import scala.concurrent.Future
+
+trait V18PsbtRpc {
+  self: Client =>
+
+  /** Analyzes and provides information about the current status of a PSBT and its inputs*/
```

Comment:
It would be a good idea to link to the bitcoin developer reference with the documentation for this rpc call

```scala
/**
Analyzes and provides information about the current status of a PSBT and its inputs
@see link to analyze psbt
*/
```